tests/nn/test_system.py

```diff
@@ -5,8 +5,39 @@
 """
 
 from sleap.nn.system import get_gpu_memory
+import os
+import pytest
 
 
 def test_get_gpu_memory():
     # Make sure this doesn't throw an exception
     memory = get_gpu_memory()
+
+
+def test_get_gpu_memory_visible():
```

Comment:
Could we also add a test (possibly it's own function/test) to check that `sleap.nn.system.get_all_gpus` returns the GPUs in the same order (ascending gpu index) as `nvidia_smi` query and same length as `sleap.nn.system.get_gpu_memory`

Reply:
done!